Code_Sample_README.md

```diff
@@ -0,0 +1,83 @@
+# blackbaud-crm-sky
+A repository containing the code needed to extend Blackbaud CRM's Fundraiser on the Go SKY-based mobile app.
+
+## Code Samples
+
+Follow these instructions to extend Blackbaud SKY-based mobile apps.
+
+### Adding logic to an existing form
+
+When completing a step/interaction, the out of the box version does not have any custom logic. If you require a category when completing a step, you can customize frogger to do the same. Here are the steps:
```

Comment:
I think it would be clearer here to speak in generic terms to set the stage for a concrete example. For instance:

"Out-of-the-box Fundraiser on the Go forms are subject to specific business rules. Sometimes, to fit your needs, you may need to override or modify these rules. For example, your organization might want to make Category a required field when completing a step or interaction. This can be done by making the following changes:"